src/robot/client.ts

```diff
@@ -114,12 +126,23 @@ export class RobotClient extends EventDispatcher implements Robot {
   constructor(
     serviceHost: string,
     webrtcOptions?: WebRTCOptions,
+    directOptions?: DirectOptions,
```

Comment:
let's make this the last option otherwise this break users who are currently creating a client via this constructor.